WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -70,13 +70,14 @@ private void InitializeTitle()
         private void BtnEquipment_Click(object sender, RoutedEventArgs e)
         {
             IContentRowsStrategy strategy = new ContentRowsStrategy(new EquipmentContentRows(mapObject.Id));
-            EquipmentAndMedicationWindow equipment = new EquipmentAndMedicationWindow(strategy.GetContentRows());
+            EquipmentAndMedicationWindow equipment = new EquipmentAndMedicationWindow(strategy.GetContentRows(), mapObject.Id);
             equipment.ShowDialog();
+            this.Close();
         }
         private void BtnMedications_Click(object sender, RoutedEventArgs e)
         {
             IContentRowsStrategy strategy = new ContentRowsStrategy(new MedicationContentRows(mapObject.Id));
-            EquipmentAndMedicationWindow medications = new EquipmentAndMedicationWindow(strategy.GetContentRows());
+            EquipmentAndMedicationWindow medications = new EquipmentAndMedicationWindow(strategy.GetContentRows(), mapObject.Id);
             medications.ShowDialog();
         }
```

Comment:
Why you close window if open equipment dialog and don't close when open medication dialog?

Reply:
I didn't pay attention, now it is in medication method too.